src/helper/docs/cheat.rs

```diff
@@ -25,6 +25,9 @@ pub fn show_cheat_sheet<Output: Write>(
         .call()
         .map_err(|e| Error::from(Box::new(e)))?
         .into_string()?;
+    if cheat_sheet.starts_with("Unknown topic.") {
+        return Err(Error::CheatSheetNotFoundError(cmd.into()));
```

Comment:
Hmm, a second thought, the suggestion text of `cheat.rs` might be helpful in case the user made a typo or there is a similar command.

```
Unknown topic.
Do you mean one of these topics maybe?
```

I recommend not aborting and printing out this message without a pager:

```diff
diff --git a/src/helper/docs/cheat.rs b/src/helper/docs/cheat.rs
index 08b25da..1f2ebda 100644
--- a/src/helper/docs/cheat.rs
+++ b/src/helper/docs/cheat.rs
@@ -26,7 +26,8 @@ pub fn show_cheat_sheet<Output: Write>(
         .map_err(|e| Error::from(Box::new(e)))?
         .into_string()?;
     if cheat_sheet.starts_with("Unknown topic.") {
-        return Err(Error::CheatSheetNotFoundError(cmd.into()));
+        writeln!(output, "{}", cheat_sheet)?;
+        return Ok(());
     }
     if let Some(pager) = pager {
         let mut process = if cfg!(target_os = "windows") {

```

So that we won't waste time with the pager if the cheat sheet isn't available and we just print out the message as-is.

What do you think? Can you change the scope of the PR like that?

Reply:
Maybe set the pager to `None` in that diff, that will do the same thing. (Going to bed now I might be unresponsive for a couple of hours.)

Reply:
I also went to bed lol, went with this approach

> Maybe set the pager to None in that diff